### PR TITLE
Move tech_name field to server env mixin + fix env override of index_prefix_name

### PR DESCRIPTION
**Pull request by simahawk**
- [x] Depends on https://github.com/OCA/server-env/pull/52

### CODE REVIEW

File: connector_search_engine/models/se_backend.py [hunk 1]
```diff
@@ -96,31 +81,17 @@ def _compute_specific_backend(self):
...
     def _onchange_tech_name(self):
-        if self.tech_name:
-            # make sure is normalized
-            self.tech_name = self._normalize_name(self.tech_name)
+        super()._onchange_tech_name()
         if self.index_prefix_name:
             # make sure is normalized
-            self.index_prefix_name = self._normalize_name(self.index_prefix_name)
+            self.index_prefix_name = self._normalize_tech_name(self.index_prefix_name)
         else:
```
Comment (i-vyshnevska) [inline connector_search_engine/models/se_backend.py:90]:
```suggestion
        elif self.tech_name:
```
as tech_name is not required anymore

Comment (simahawk) [reply connector_search_engine/models/se_backend.py:90]: it is https://github.com/OCA/server-env/pull/52/files#diff-9fb128f0d1213751a2f5f5c5f3c3c382R36

**Comment by guewen** [comment]
Do you know why the tests are choking trying to connect to an elasticsearch service?

**Comment by guewen** [comment]
> Do you know why the tests are choking trying to connect to an elasticsearch service?

Could it be that the cassettes are matched against the host and the server env config changes the host?

**Comment by simahawk** [comment]
tests are fixed here https://github.com/OCA/search-engine/pull/50 and I don't see any change in the URLs.

**Review by guewen — APPROVED** [review]
pre-approving as long as the tests are fixed in #50

**Review by i-vyshnevska — APPROVED** [review]

**Review by lmignon — APPROVED** [review]

**Pull request merged**